python/pypto/ir/directions.py

```
@@ -0,0 +1,99 @@
+
+
+__all__ = [
+    "ArgDirection",
+    "input",
+    "output",
+    "output_existing",
+    "inout",
+    "no_dep",
+    "scalar",
+    "make_call",
+]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)